castle/cms/static/plone-compiled.js

```diff
@@ -30165,5 +30167,5 @@ $(document).ready(function(){
 
 });
 
-define("/Users/brian.duncan/castle-instances/pages-to-folders/castle/cms/static/plone.js", function(){});
+define("/Users/katiewiessfelt/dev/git/castle/fbigov-dev/src/castle.cms/castle/cms/static/plone.js", function(){});
```

Comment:
I wonder if there is a way to remove the specific paths for these from the compiled file?

Reply:
I couldn't find an easy way to do this, but I can keep looking into it.